src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/config/PaymentConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.infrastructure.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableConfigurationProperties
+@ConfigurationProperties("payment.params")
+public class PaymentConfiguration {
```

Comment:
could you change class name for `PaymentProperties`. I think `Configuration` name should be reserved for place where we define Beans only. It supposes to not be injected in regular Service Beans